### PR TITLE
Adding 125 Valid palindrome in Python

### PR DESCRIPTION
**Pull request by vale-c**
The purpose of this PR is to add my solution to the `Valid Palindrome` Leetcode problem which can be found [here](https://leetcode.com/problems/valid-palindrome/).

### SCREENSHOT 📸 - (Preview of the Solution page)

0125 - Valid Palindrome (Easy) - LeetCode The Hard Way - localhost

### CODE REVIEW

File: solutions/0100-0199/0125-valid-palindrome.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+
+We can use two pointers to check if the string is a palindrome. We start from the beginning and the end of the string and move progressively towards the middle of the string. 
+
+If the characters at the two pointers are *not* the same, we return *false*. 
+
+If we reach the middle of the string, we return *true*.
+
+<SolutionAuthor name="@vale-c"/>
+
+```python
```
Comment (wingkwong) [inline solutions/0100-0199/0125-valid-palindrome.md:33]: Use this format.

````
<Tabs>
<TabItem value="py" label="Python">
<SolutionAuthor name="@yourname"/>

```py
# Your code goes here
```

</TabItem>
</Tabs>
````

File: solutions/0100-0199/0125-valid-palindrome.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+Given a string `s`, return `true` if it is a palindrome, or `false` otherwise.
+
+**Constraints:**
+
+* `1 <= s.length <= 2 * 105`
+* `s` consists only of printable ASCII characters.
+
+
+## Approach: Two Pointers
```
Comment (wingkwong) [inline solutions/0100-0199/0125-valid-palindrome.md:23]: add time complexity & space complexity of your solution

File: solutions/0100-0199/0125-valid-palindrome.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+If the characters at the two pointers are *not* the same, we return *false*. 
+
+If we reach the middle of the string, we return *true*.
+
+<SolutionAuthor name="@vale-c"/>
+
+```python
+class Solution:
+    def isPalindrome(self, s: str) -> bool:
+        alpha = ''.join(char for char in s.lower() if char.isalnum())
```
Comment (wingkwong) [inline solutions/0100-0199/0125-valid-palindrome.md:36]: explain what this line does for beginners

File: solutions/0100-0199/0125-valid-palindrome.md [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+        
+        start = 0
+        end = len(alpha)-1
+        
+        if s == ' ':
+            return True
```
Comment (wingkwong) [inline solutions/0100-0199/0125-valid-palindrome.md:42]: these 2 lines are unncessary.

**Review by wingkwong — CHANGES_REQUESTED** [review]

File: solutions/0100-0199/0125-valid-palindrome.md (hunk 2 above)
Comment (vale-c) [reply solutions/0100-0199/0125-valid-palindrome.md:22]: Good point! I have just added both time/space complexity for this algorithm ✨

File: solutions/0100-0199/0125-valid-palindrome.md (hunk 4 above)
Comment (vale-c) [reply solutions/0100-0199/0125-valid-palindrome.md]: Nice catch! 💯

File: solutions/0100-0199/0125-valid-palindrome.md (hunk 3 above)
Comment (vale-c) [reply solutions/0100-0199/0125-valid-palindrome.md]: I have added an explanation both as a comment in the code and in the description above.

**Review by wingkwong — APPROVED** [review]
Overall ok. I'll fine tune a bit myself. Thanks.

**Pull request merged**